libraries/AP_NavEKF3/AP_NavEKF3_AirDataFusion.cpp

```diff
@@ -444,6 +449,318 @@ void NavEKF3_core::FuseSideslip()
+            const float HK13 = 2*HK11;
+            const float HK14 = 2*HK4;
+            const float HK15 = 2*HK2;
+            const float HK16 = 2*HK5;
+            const float HK17 = 2*HK6;
+            const float HK18 = -HK12*P[0][23] + HK12*P[0][5] - HK13*P[0][6] + HK14*P[0][1] + HK15*P[0][0] - HK16*P[0][2] + HK17*P[0][3] - HK7*P[0][22] + HK7*P[0][4];
+            const float HK19 = HK12*P[5][23];
+            const float HK20 = -HK12*P[23][23] - HK13*P[6][23] + HK14*P[1][23] + HK15*P[0][23] - HK16*P[2][23] + HK17*P[3][23] + HK19 - HK7*P[22][23] + HK7*P[4][23];
+            const float HK21 = powf(Kacc, 2);
+            const float HK22 = HK12*HK21;
+            const float HK23 = HK12*P[5][5] - HK13*P[5][6] + HK14*P[1][5] + HK15*P[0][5] - HK16*P[2][5] + HK17*P[3][5] - HK19 + HK7*P[4][5] - HK7*P[5][22];
+            const float HK24 = HK12*P[5][6] - HK12*P[6][23] - HK13*P[6][6] + HK14*P[1][6] + HK15*P[0][6] - HK16*P[2][6] + HK17*P[3][6] + HK7*P[4][6] - HK7*P[6][22];
+            const float HK25 = HK7*P[4][22];
+            const float HK26 = -HK12*P[4][23] + HK12*P[4][5] - HK13*P[4][6] + HK14*P[1][4] + HK15*P[0][4] - HK16*P[2][4] + HK17*P[3][4] - HK25 + HK7*P[4][4];
+            const float HK27 = HK21*HK7;
+            const float HK28 = -HK12*P[22][23] + HK12*P[5][22] - HK13*P[6][22] + HK14*P[1][22] + HK15*P[0][22] - HK16*P[2][22] + HK17*P[3][22] + HK25 - HK7*P[22][22];
+            const float HK29 = -HK12*P[1][23] + HK12*P[1][5] - HK13*P[1][6] + HK14*P[1][1] + HK15*P[0][1] - HK16*P[1][2] + HK17*P[1][3] - HK7*P[1][22] + HK7*P[1][4];
+            const float HK30 = -HK12*P[2][23] + HK12*P[2][5] - HK13*P[2][6] + HK14*P[1][2] + HK15*P[0][2] - HK16*P[2][2] + HK17*P[2][3] - HK7*P[2][22] + HK7*P[2][4];
+            const float HK31 = -HK12*P[3][23] + HK12*P[3][5] - HK13*P[3][6] + HK14*P[1][3] + HK15*P[0][3] - HK16*P[2][3] + HK17*P[3][3] - HK7*P[3][22] + HK7*P[3][4];
+            // const float HK32 = Kacc/(-HK13*HK21*HK24 + HK14*HK21*HK29 + HK15*HK18*HK21 - HK16*HK21*HK30 + HK17*HK21*HK31 - HK20*HK22 + HK22*HK23 + HK26*HK27 - HK27*HK28 + R_ACC);
+
+            // calculate innovation variance and exit if badly conditioned
+            innovDragVar.x = (-HK13*HK21*HK24 + HK14*HK21*HK29 + HK15*HK18*HK21 - HK16*HK21*HK30 + HK17*HK21*HK31 - HK20*HK22 + HK22*HK23 + HK26*HK27 - HK27*HK28 + R_ACC);
+            if (innovDragVar.x < R_ACC) {
+                return;
+            }
+            const float HK32 = Kacc / innovDragVar.x;
+
+            // Observation Jacobians
+            Hfusion[0] = -HK2*HK3;
+            Hfusion[1] = -HK3*HK4;
+            Hfusion[2] = HK3*HK5;
+            Hfusion[3] = -HK3*HK6;
+            Hfusion[4] = -HK8;
+            Hfusion[5] = -HK10;
+            Hfusion[6] = HK11*HK3;
+            Hfusion[22] = HK8;
+            Hfusion[23] = HK10;
+
+            // Kalman gains
+            // Don't allow modification of any states other than wind velocity - we only need a wind estimate.
```

Comment:
Do we want to leave all this commented out code in?

Reply:
I've replaced it with a link to the relevant derivation output file.